index.html

```diff
@@ -69,5 +70,185 @@ <h2 class="text-2">
         
     </section>
 
+    <section class="wrapper-works">
+        <header class="title">
+            My Recent Works
+        </header>
+        
+        
+        <img src="./images/img-work-mobile.png" class="image-work">
+
+        <h3 class="title-multi">
+            Multi-Post Stories
+        </h3>
+
+        <p class="works-description">
+            A daily selection of privately personalized 
+                reads; no accounts or sign-ups required. 
+                has been the industry's standard dummy text
+                ever since the 1500s, when an unknown
+                printer took a standard dummy text.
+        </p>
+
+        
+        <ul class="button-container">
+            <li class="button1">css</li>
+            <li class="button1">html</li>
+            <li class="button1">bootstrap</li>
+            <li class="button1">Ruby</li> 
+            <button class="button2">See Project</button>                                  
+        </ul>   
+
+    </section>
+
+    <div class="wrapper-cards">
+
+        <div class="card card-1">
+            <h2 class="title-post"> Profesional Art Printing Data </h2>
+            <p class="supporting-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+
+            <ul class="button-container-float">
+               <li class="button3">html</li>
+               <li class="button3">bootstrap</li>
+               <li class="button3">Ruby</li>                                                    
+            </ul>            
+           <div class="orange-button">
+               See Project
+           </div>                                
```

Comment:
- [ ] As the previous reviewer requested, please update every button to be an `<a>` tag or a `<button>` tag. This is because it will define better the purpose of this element to the browser, and improve semantics as well as accessibility! 🤓 🛠️